MANIFEST.in

```diff
@@ -0,0 +1,13 @@
+prune *
```

Comment:
Seems this `MANIFEST.in` is still including too much things, specially the whole `src` of the webui, which isn't needed. Please, include only `raiden_webui` directory (it is, `__init__.py` plus `ui` dist folder)

Reply:
The `src` should not be included. `prune` is supposed to exclude all files under the specified directory. I am probably doing something wrong though because I have to specifically exclude some files in the root directory otherwise they are packaged.

In the second line I recursively include the `raiden_webui` directory. 

At least when building the package locally it didn't include `src`, if the experience was different for you could you provide more information about it?